### DISCUSSION TITLE
How to backup and restore database in native install

### DISCUSSION THREAD

**Discussion by singalsu** [teslamate-org/teslamate#2615] [Getting started] [answered]
I think my Teslamate upgrade attempt failed. I don't have new data in Grafana since about one week.

The guide for backup and restore is only for container setup. How can I do the same in native install in Ubuntu?

```
# exec -T database pg_dump -U teslamate teslamate > /mnt/backup/teslamate.bck
bash: exec: -T: invalid option
exec: usage: exec [-cl] [-a name] [command [argument ...]] [redirection ...]

```

Thanks!

**Comment by cwanja** [comment, marked as answer]
https://vegastack.com/tutorials/how-to-backup-and-restore-postgresql-database/#restoring-a-postgresql-database

**Reply by singalsu** [reply]
Thanks @cwanja !
